SDK_versioned_docs/version-3.0.0/subgraph-example.md

```diff
@@ -0,0 +1,282 @@
+---
+id: Subgraph examples
+sidebar_position: 3
```

Comment:
can we put this doc in a folder called 'Subgraph' and include the 'Uniswap Subgraph Data' doc in that folder too?

let's put this folder after the Guides folder